docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -2649,6 +2649,71 @@ class DetailResource {
 
 WARNING: Unlike with `@Inject` the templates obtained via `RestTemplate` are not validated, i.e. the build does not fail if a template does not exist.
 
+[[vertx_integration]]
+=== Vertx Integration
+
+If you want to use `io.vertx.core.json.JsonObject` as data in your templates, then you will need to add the `quarkus-vertx` extension to your `pom.xml` file (most applications use this extension by default).
```

Comment:
```suggestion
If you want to use `io.vertx.core.json.JsonObject` as data in your templates, then you will need to add the `quarkus-vertx` extension to your build file if not already part of your dependencies (most applications use this extension by default).
```

Reply:
It would probably make sense to remove this paragraph completely.